gtsam/geometry/Cal3Fisheye.cpp

```diff
@@ -110,21 +103,19 @@ Point2 Cal3Fisheye::uncalibrate(const Point2& p, OptionalJacobian<2, 9> H1,
 /* ************************************************************************* */
 Point2 Cal3Fisheye::calibrate(const Point2& uv, OptionalJacobian<2, 9> Dcal,
                               OptionalJacobian<2, 2> Dp) const {
-  // Apply inverse camera matrix to map the pixel coordinate (u, v) 
+  // Apply inverse camera matrix to map the pixel coordinate (u, v)
   // of the equidistant fisheye image to angular coordinate space (xd, yd)
   // with radius theta given in radians.
   const double u = uv.x(), v = uv.y();
   const double yd = (v - v0_) / fy_;
   const double xd = (u - s_ * yd - u0_) / fx_;
-  const double theta = sqrt(xd * xd + yd * yd);
-  
-  // Provide initial guess for the Gauss-Newton search.
+
+  // Provide initial guess for the Newton search.
   // The angular coordinates given by (xd, yd) are mapped back to
   // the focal plane of the perspective undistorted projection pi.
-  // See Cal3Unified.calibrate() using the same pattern for the 
+  // See Cal3Unified.calibrate() using the same pattern for the
   // undistortion of omnidirectional fisheye projection.
-  const double scale = (theta > 0) ? tan(theta) / theta : 1.0;
```

Comment:
this scale don't need!
because 
theta_d = sqrt(xd * xd + yd * yd) in (31)
theta = theta_d inverse in (32)
![image](https://user-images.githubusercontent.com/7278867/201642119-b9f07788-43ca-42b2-811e-63e0064a78c6.png)